src/Ionide.ProjInfo/Library.fs

```diff
@@ -1224,7 +1214,7 @@ type WorkspaceLoaderViaProjectGraph private (toolsPath, ?globalProperties: (stri
 
                 let bm = BuildManager.DefaultBuildManager
                 use sw = new StringWriter()
-                let loggers = ProjectLoader.createLoggers allKnownNames binaryLogs sw
+                let loggers = ProjectLoader.createLoggers "graph-build" binaryLogs sw
```

Comment:
Previously we'd create a binlog _for each project_ that we were design time building. This would cause excessive slowdowns.